app/web/features/communities/EditCommunityInfoPage.tsx

```diff
@@ -148,11 +148,6 @@ export default function EditCommunityPage({
                 imageUpload
                 required={t("communities:page_content_required")}
               />
-              {errors.content && (
-                <Typography color="error" variant="body2">
-                  {errors.content.message}
-                </Typography>
-              )}
```

Comment:
I moved the error functionality to be within the MarkdownInput since we are passing the errorText in multiple components already but not displaying it.

Now that that's done I can remove this extra error logic.